install-gemfile-lock

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
Minor: my understanding is that `#!/bin/bash` is preferred because the above makes it a little easier to spoof bash.

Reply:
I think using env makes the script more portable though because individuals might be using bash installed at locations, for example I always want script to use Bash v4 located at `/opt/homebrew/bin/bash` on my machine.

Reply:
env is preferred imo